ggshield/verticals/sca/file_selection.py

```diff
@@ -93,4 +98,11 @@ def sca_files_from_git_repo(
             raise APIKeyCheckError(client.base_uri, "Invalid API key.")
         raise UnexpectedError("Failed to select SCA files")
 
+    sca_files = sca_files_result.sca_files
+    if verbose:
+        state_name = "previous" if previous_state else "current"
+        display_info(f"> Scanned files from {state_name} state:")
```

Comment:
As what is the previous state is not that clear depending on the scan, I would print `ref` instead, and delete the bool `previous_state` from the args:
```suggestion
        display_info(f"> Scanned files from {ref}:")
```